Sources/swift-build-prebuilts/BuildPrebuilts.swift

```diff
@@ -0,0 +1,331 @@
+                        try fm.createDirectory(atPath: modulesDir.pathString, withIntermediateDirectories: true)
+                        try fm.createDirectory(atPath: includesDir.pathString, withIntermediateDirectories: true)
+
+                        // Clean out the scratch dir
+                        if fm.fileExists(atPath: scratchDir.pathString) {
+                            try fm.removeItem(atPath: scratchDir.pathString)
+                        }
+
+                        // Build
+                        var cmd = ""
+                        if docker, let dockerTag = platform.dockerTag, let dockerPlatform = platform.arch.dockerPlatform {
+                            cmd += "\(dockerCommand) run --rm --platform \(dockerPlatform) -v \(repoDir):\(repoDir) -w \(repoDir) \(dockerImageRoot)\(dockerTag) "
+                        }
+                        cmd += "swift build -c release --arch \(platform.arch) --product \(library.name)"
+                        try await shell(cmd)
+
+                        // Copy the library to staging
+                        let lib = "lib\(library.name).a"
+                        try fm.copyItem(atPath: buildDir.appending(lib).pathString, toPath: libDir.appending(lib).pathString)
+
+                        // Copy the swiftmodules
+                        for file in try fm.contentsOfDirectory(atPath: srcModulesDir.pathString) {
+                            try fm.copyItem(atPath: srcModulesDir.appending(file).pathString, toPath: modulesDir.appending(file).pathString)
+                        }
+
+                        // Copy the C module headers
+                        for cModule in library.cModules {
+                            let cModuleDir = version.cModulePaths[cModule] ?? ["Sources", cModule]
+                            let srcIncludeDir = repoDir.appending(components: cModuleDir).appending("include")
+                            let destIncludeDir = includesDir.appending(cModule)
+                            try fm.createDirectory(atPath: destIncludeDir.pathString, withIntermediateDirectories: true)
+                            for file in try fm.contentsOfDirectory(atPath: srcIncludeDir.pathString) {
+                                try fm.copyItem(atPath: srcIncludeDir.appending(file).pathString, toPath: destIncludeDir.appending(file).pathString)
+                            }
+                        }
+
+                        // Zip it up
+                        _ = fm.changeCurrentDirectoryPath(stageDir.pathString)
+                        let zipFile = stageDir.appending("\(swiftVersion)-\(library.name)-\(platform).zip")
+                        let contentDirs = ["lib", "Modules"] + (library.cModules.isEmpty ? [] : ["include"])
+#if os(Windows)
+                        try await shell("tar -acf \(zipFile.pathString) \(contentDirs.joined(separator: " "))")
+#else
+                        try await shell("zip -r \(zipFile.pathString) \(contentDirs.joined(separator: " "))")
```

Comment:
(non-blocking): Why not just use `tar` everywhere? I think it's more commonly installed by default than zip is.